src/lib/modules/Loan/backoffice/OverdueLoanSendMailModal/state/actions.js

```diff
@@ -20,12 +23,15 @@ export const sendOverdueLoansMailReminder = (loanPid) => {
         type: SUCCESS,
         payload: response.data,
       });
-      dispatch(
-        sendSuccessNotification(
-          'Success!',
-          'An email is on its way to the user.'
-        )
-      );
+      setTimeout(() => {
```

Comment:
Here the success action is dispatched but the mail has not been send yet (it's not in the db) so I set a 5 sec timeout until the email is sent. Is there any better solution?